huaweicloud/services/css/resource_huaweicloud_css_cluster.go

```diff
@@ -0,0 +1,680 @@
+package css
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/chnsz/golangsdk"
+	"github.com/chnsz/golangsdk/openstack/css/v1/cluster"
+	"github.com/chnsz/golangsdk/openstack/css/v1/snapshots"
+	"github.com/hashicorp/go-multierror"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/common"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/config"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/utils"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/utils/fmtp"
+)
+
+func ResourceCssCluster() *schema.Resource {
```

Comment:
Suggest adding version of API to the function name.

Reply:
Suggest this resource  needs to shield the version of api. so keep it as it is